web-server/src/orm/entities/users/Files.ts

```diff
@@ -0,0 +1,25 @@
+import bcrypt from 'bcryptjs';
+import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, UpdateDateColumn } from 'typeorm';
+
+@Entity('users')
+export class User {
```

Comment:
This should be File rather than User. Also change the table name from `users` to `files`.